docker/server/configs/php.ini

```diff
@@ -0,0 +1,1940 @@
+[PHP]
```

Comment:
This file is too big, can we please remove unnecessary configurations and comments and keep it clean. I don't think we require all of these configs

Reply:
sure thing.